### DISCUSSION TITLE
Can I use FontAwesome icons in GitHub flavored markdown?

### DISCUSSION THREAD

**Discussion by sr-murthy** [FortAwesome/Font-Awesome#20280] [Q&A]
Can I use FontAwesome icons in GitHub flavored markdown?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

You mean here on GitHub?

**Reply by sr-murthy** [reply]
Yes, like say in a CONTRIBUTIONS.md in a repo.

**Reply by tagliala** [reply]
I think it is not possible, I would use emojis

**Reply by sr-murthy** [reply]
Thought so. Np, thanks.
